SH4RE/SH4RE/Account/AccountView.swift

```diff
@@ -36,14 +38,27 @@ struct AccountView: View {
                         Text("Logout")
                     }
                     .buttonStyle(secondaryButtonStyle())
+                    
+                    //Favourites
+                    VStack (alignment: .leading) {
```

Comment:
can you remove changes in this file for now, I will add a part of account page ticket

Reply:
Ill remove most of them, but I am gonna leave the passing of favourites model